OpenRA.Mods.Common/Traits/SupportPowers/SupportPower.cs

```diff
@@ -175,5 +177,17 @@ public virtual void PlayLaunchSounds()
 			var speech = isAllied ? Info.LaunchSpeechNotification : Info.IncomingSpeechNotification;
 			Game.Sound.PlayNotification(Self.World.Map.Rules, toPlayer, "Speech", speech, toPlayer.Faction.InternalName);
 		}
+
+		public IEnumerable<CPos> CellsMatching(CPos location, string footprint, CVec dimensions)
+		{
+			var index = 0;
+			var fp = footprint.Where(c => !char.IsWhiteSpace(c)).ToArray();
```

Comment:
Maybe store this as the footprint field so we don't need to do this each time.

Reply:
Good idea. Done.